### PR TITLE
fix the typo of errorf info

### PR DESCRIPTION
**Pull request by jianglingxia**
**What this PR does / why we need it**:
fix the error message of stateful_pod_control_test.go
**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #

**Special notes for your reviewer**:

**Release note**:

```release-note
NONE
```

### CODE REVIEW

**Comment by jianglingxia** [comment]
PTAL @janetkuo
/assign @enisoc

**Comment by jianglingxia** [comment]
PTAL @janetkuo
/assign @enisoc

**Comment by jianglingxia** [comment]
PTAL @foxish
/assign @foxish

**Comment by jianglingxia** [comment]
PTAL @janetkuo
/assign @enisoc

File: pkg/controller/statefulset/stateful_pod_control_test.go [hunk 1]
```diff
@@ -63,7 +63,7 @@ func TestStatefulPodControlCreatesPods(t *testing.T) {
 	}
 	for i := range events {
 		if !strings.Contains(events[i], v1.EventTypeNormal) {
-			t.Errorf("Expected normal events found %s", events[i])
+			t.Errorf("Expected normal event found %s", events[i])
```
Comment (enisoc) [inline pkg/controller/statefulset/stateful_pod_control_test.go]: The original wording here is weird, particularly the missing punctuation before "found". While you're at it, would you mind updating the general style of these error messages? That would make it more worthwhile to go through the process of review and presubmit checks.

For example, I think this error would be more clear as: `"found unexpected non-normal event: %s"`

File: pkg/controller/statefulset/stateful_pod_control_test.go [hunk 2]
```diff
@@ -93,7 +93,7 @@ func TestStatefulPodControlCreatePodExists(t *testing.T) {
 	}
 	events := collectEvents(recorder.Events)
 	if eventCount := len(events); eventCount != 0 {
-		t.Errorf("Expected 0 events when Pod and PVC exist found %d", eventCount)
+		t.Errorf("Expected 0 event when Pod and PVC exist found %d", eventCount)
```
Comment (enisoc) [inline pkg/controller/statefulset/stateful_pod_control_test.go]: You could update these to better fit the conventional Go style of "actual" before "expected":

https://github.com/golang/go/wiki/CodeReviewComments#useful-test-failures

For example: `"Pod and PVC exist: got %d events, want 0"`

File: pkg/controller/statefulset/stateful_pod_control_test.go [hunk 3]
```diff
@@ -124,7 +124,7 @@ func TestStatefulPodControlCreatePodPvcCreateFailure(t *testing.T) {
 	}
 	for i := range events {
 		if !strings.Contains(events[i], v1.EventTypeWarning) {
-			t.Errorf("Expected normal events found %s", events[i])
+			t.Errorf("Expected normal event found %s", events[i])
```
Comment (enisoc) [inline pkg/controller/statefulset/stateful_pod_control_test.go]: The original message seems inaccurate. According to the code it should have said `"found unexpected non-warning event: %s"`

**Comment by jianglingxia** [comment]
PTAL @enisoc
I have updated it ,thanks for your review again!

**Pull request merged**